snippets/ModalDialog.md

```diff
@@ -0,0 +1,135 @@
+### Modal/Dialog
+
+Renders a dialog component with modal.
+
+This is a simple React ModalDialog Component.It owns only one prop: `closeOnClick`.
```

Comment:
We should really move this to state. I've tested it on StackBlitz and it works as expected so I'm going to suggest it again

```suggestion
This is a simple React ModalDialog Component that is called and controlled through events
```